homeassistant/components/livisi/climate.py

```diff
@@ -0,0 +1,214 @@
+"""Code to handle a Livisi Virtual Climate Control."""
```

Comment:
Please add tests or add climate.py to the coverage ignore file `.coveragerc`

Reply:
I added the tests from climate.py to the coverage ignore file. Thank you.